### PR TITLE
Redefine original in manifest(ManagedReference only temporary)

### PR DESCRIPTION
**Pull request by qinezh**
@chenkennt @vwxyzh @superyyrrzz @hellosnow @ansyral @herohua

### CODE REVIEW

**Comment by qinezh** [comment]
@herohua

File: src/Microsoft.DocAsCode.Build.ManagedReference/ManagedReferenceDocumentProcessor.cs [hunk 1]
```diff
@@ -79,9 +79,18 @@ public override FileModel Load(FileAndType file, ImmutableDictionary<string, obj
                         }
                     }
                     var filePath = Path.Combine(file.BaseDir, file.File);
-                    var repoInfo = GitUtility.GetGitDetail(filePath);
-                    // Item's source is the path for the original code, should not be used here
-                    var displayLocalPath = repoInfo?.RelativePath ?? filePath.ToDisplayPath();
+                    var displayLocalPath = filePath.ToDisplayPath();
+
+                    object rootRepoInfo;
+                    if (metadata.TryGetValue("newFileRepository", out rootRepoInfo))
```
Comment (herohua) [inline src/Microsoft.DocAsCode.Build.ManagedReference/ManagedReferenceDocumentProcessor.cs]: where is this value set?

**Comment by qinezh** [comment]
@herohua @superyyrrzz Updated

**Comment by herohua** [comment]
(LGTM)

**Comment by herohua** [comment]
:+1:

**Comment by vwxyzh** [comment]
👍

**Pull request merged**